gradle/dependencies.gradle

```diff
@@ -6,10 +6,18 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
     testImplementation 'org.mockito:mockito-inline:4.5.1'
     
-    api 'com.google.code.gson:gson:2.9.0'
+    implementation 'com.google.code.gson:gson:2.9.0'
```

Comment:
do we need to have a dependency on gson here anymore? I believe it's going to be brought by the serialization package.

Reply:
correct

Reply:
```suggestion
```